test/periphery/CardPaymentProcessorUpgradeable.test.ts

```diff
@@ -0,0 +1,1415 @@
+import { ethers, upgrades } from "hardhat";
+import { expect } from "chai";
+import { BigNumber, Contract, ContractFactory } from "ethers";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/dist/src/signer-with-address";
+import { proveTx } from "../../test-utils/eth";
+import { createBytesString } from "../../test-utils/misc";
+
+const BYTES16_LENGTH: number = 16;
+const BYTES32_LENGTH: number = 32;
+const ZERO_BYTES16_STRING: string = createBytesString('00', BYTES16_LENGTH);
+const ZERO_BYTES32_STRING: string = createBytesString('00', BYTES32_LENGTH);
+const SOME_BYTES16_STRING = createBytesString('ABCD', BYTES16_LENGTH);
+const SOME_BYTES32_STRING = createBytesString('DCBA', BYTES32_LENGTH);
+
+enum PaymentStatus {
+  Nonexistent = 0,
+  Uncleared = 1,
+  Cleared = 2,
+  Revoked = 3,
+  Reversed = 4,
+  Confirmed = 5,
+}
+
+interface TestPayment {
+  authorizationId: number,
+  account: SignerWithAddress,
+  amount: number,
```

Comment:
Codacy has a fix for the issue: [Replace `,` with `;`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9781596)
```suggestion
  amount: number;
```